packages/rancher-istio/charts/templates/istio-psp.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: istio-psp
```

Comment:
Wondering if we should expose this in `values.yaml`
(and its references throughout the other file(s))